### ISSUE TITLE
ngrok, Django 2.1 and SESSION_COOKIE_SAMESITE 

### ISSUE THREAD

**Issue by bianchimro** [discolabs/django-shopify-auth#47]
Hi and thanks a lot for sharing the package.
I am developing on the latest django (2.1) and I am using `ngrok` to develop locally, following your example for `settings.py` (except for the `TEMPLATE_CONTEXT_PROCESSORS` setting that now lives under the `TEMPLATES`)

I got stuck because my app didn't seem to authenticate properly.

After some hours of research I found that a new setting "SESSION_COOKIE_SAMESITE" which is new in django 2.1 prevented the session-based login to work, resulting in an infinite redirect loop.

To recap: with django 2.1+ you must set
```
SESSION_COOKIE_SAMESITE = False
```

Probably this should be mentioned in the README

Best regards
Mauro

**Comment by timonweb** [comment]
@bianchimro you've just saved so much hair on my head, I've been banging against the wall for several straight days.... gosh.... thank you so MUCH!

**Comment by pirsquare** [comment]
In [Django 3.1](https://docs.djangoproject.com/en/3.1/ref/settings/#session-cookie-samesite), use `SESSION_COOKIE_SAMESITE = "None"`. Note that `None` here is a string.
